hw-7/7.1/style.css

```diff
@@ -0,0 +1,120 @@
+* {
```

Comment:
In general, good job for Namaste project. You did it it that way which I would use